src/components/Pages/Unit/widgets/UnitMaterials/UnitBar.js

```diff
@@ -3,13 +3,17 @@ import { Button } from 'semantic-ui-react';
 import PropTypes from 'prop-types';
 
 import LibrarySettings from '../../../../Sections/Library/LibrarySettings';
+import BookmarkButton from '../../../../shared/SaveBookmark/BookmarkButton';
 
-const UnitBar = ({ fontSize = 0, handleSettings }) => {
+const UnitBar = ({ fontSize = 0, handleSettings, source }) => {
 
   const print = () => window.print();
 
   return (
     <div className="source__header-toolbar">
+      {
```

Comment:
![image](https://user-images.githubusercontent.com/1877419/146888706-84c0293f-282f-4c91-8f17-c636cd086907.png)

In mobile icon overflows title of source.

Reply:
its separated bug